BaleLoaderAIDriver.lua

```diff
@@ -72,6 +72,14 @@ function BaleLoaderAIDriver:init(vehicle)
 	self:debug('Initialized, bale loader: %s', self.baleLoader:getName())
 end
 
+function BaleLoaderAIDriver:writeUpdateStream(streamId)
```

Comment:
These aren't needed as they are derived from AIDriver and we don't override the base class functions.